src/scripts/main.js

```diff
@@ -1,7 +1,92 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+function initializeGame() {
+  const boardElement = document.querySelector('.game-field');
+  const scoreElement = document.querySelector('.game-score');
+  const button = document.querySelector('.start');
+  const messageStart = document.querySelector('.message-start');
+  const messageLose = document.querySelector('.message-lose');
+  const messageWin = document.querySelector('.message-win');
+
+  function renderBoard() {
+    boardElement.innerHTML = '';
+
+    game.getState().forEach((row) => {
+      const rowElement = document.createElement('tr');
+
+      rowElement.classList.add('field-row');
+
+      row.forEach((cell) => {
+        const cellElement = document.createElement('td');
+
+        cellElement.classList.add('field-cell');
+
+        if (cell !== 0) {
+          cellElement.classList.add(`field-cell--${cell}`);
+          cellElement.textContent = cell;
+        }
+        rowElement.appendChild(cellElement);
+      });
+      boardElement.appendChild(rowElement);
+    });
+  }
+
+  function renderScore() {
+    scoreElement.textContent = game.getScore();
+  }
+
+  function renderStatus() {
+    messageStart.classList.toggle('hidden', game.getStatus() !== 'idle');
+    messageLose.classList.toggle('hidden', game.getStatus() !== 'lose');
+    messageWin.classList.toggle('hidden', game.getStatus() !== 'win');
+  }
+
+  button.addEventListener('click', () => {
+    const buttonText = button.textContent;
+
+    if (buttonText === 'Start') {
+      game.start();
+      button.textContent = 'Restart';
+      button.classList.replace('start', 'restart');
+    } else {
+      game.restart();
+      renderScore(0);
+      button.textContent = 'Start';
+      button.classList.replace('restart', 'start');
+    }
+    renderBoard();
+    renderScore();
+    renderStatus();
+  });
+
+  document.addEventListener('keydown', (keyEvent) => {
+    const key = keyEvent.key;
+
+    if (['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown'].includes(key)) {
+      keyEvent.preventDefault();
+
+      switch (key) {
+        case 'ArrowLeft':
```

Comment:
Create a variable for arrows string and use it everywhere

Reply:
tests are run locally
![test (2)](https://github.com/user-attachments/assets/47d9eb6a-14ab-4bf0-b330-3f91ed85c64f)